cognite/neat/core/extractors/graph_sheet_to_graph.py

```diff
@@ -0,0 +1,132 @@
+import logging
+import warnings
+
+import pandas as pd
+from rdflib import RDF, Graph, Literal
+
+from cognite.neat.core.data_classes.transformation_rules import TransformationRules
+
+
+def validate_if_graph_capturing_sheet_empty(graph_capturing_sheet: dict[str, pd.DataFrame]):
+    """Validate if the graph capturing sheet is empty
+
+    Parameters
+    ----------
+    graph_capturing_sheet : dict[str, pd.DataFrame]
+        Graph capturing sheet
+    """
+    if all(df.empty for df in graph_capturing_sheet.values()):
+        logging.error("Graph capturing sheet is empty! Aborting!")
+        raise ValueError("Graph capturing sheet is empty! Aborting!")
+
+
+def validate_rules_graph_pair(graph_capturing_sheet: dict[str, pd.DataFrame], transformation_rule: TransformationRules):
+    """Validate if the graph capturing sheet is based on the transformation rules
+
+    Parameters
+    ----------
+    graph_capturing_sheet : dict[str, pd.DataFrame]
+        Graph capturing sheet
+    transformation_rule : TransformationRules
+        Transformation rules
+    """
+    intersection = set(graph_capturing_sheet.keys()).intersection(set(transformation_rule.get_defined_classes()))
+
+    if not intersection:
+        logging.error("Graph capturing sheet is not based on transformation rules! Aborting!")
+        raise ValueError("Graph capturing sheet is not based on transformation rules! Aborting!")
```

Comment:
There are a lot of duplicated messages here. This will cause more work if the wording needs to change, and risk having inconsistency between the log and the ValueErrors/Warnings. Suggest you do the fix I shown here for all the clauses. 

```suggestion
        msg = "Graph capturing sheet is not based on transformation rules! Aborting!"
        logging.error(msg)
        raise ValueError(msg)
```

Reply:
Done